config/default/field.storage.node.field_h_html_content.yml

```diff
@@ -1,18 +1,18 @@
-uuid: 59d4971d-bbd9-4b26-97fb-5a4c46377458
+uuid: 0ccc5611-596d-40b8-98ec-df262579688e
 langcode: en
 status: true
 dependencies:
   module:
-    - paragraphs
+    - node
     - text
-id: paragraph.field_h_resource_file_summary
-field_name: field_h_resource_file_summary
-entity_type: paragraph
+id: node.field_h_html_content
+field_name: field_h_html_content
+entity_type: node
 type: text_long
```

Comment:
Sorry @Tjocksmurf this one is on me. Didn't realise that the HTML field should be a paragraphs field instead of an longtext. Could you please update this field so that it is the same as the HTML content field in HGA7 (https://health.govcms.gov.au/admin/structure/types/manage/publication/fields).